src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -378,11 +384,19 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
         osd.stats_history.in_bytes = osd.stats_history.op_in_bytes.map((i: string) => i[1]);
         osd.stats.usage = osd.stats.stat_bytes_used / osd.stats.stat_bytes;
         osd.cdIsBinary = true;
+        this.get_osd_device_count(osd.id);
+        osd.device_count = this.device_count;
```

Comment:
Not needed. Objects can be modified within JS functions when they are passed as function parameters.
```suggestion
```